Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -1807,9 +2531,14 @@ def main():
         LOG(f'Command being called is {command}')
         if command in commands.keys():
             commands[command]()
+        elif command in commands_auth_code.keys():
+            validate_auth_code_flow_params(command)  # raises error in case one of the required params is missing
```

Comment:
[in](https://docs.python.org/reference/expressions.html#membership-test-operations) tests for the existence of a key in a dict
```suggestion
        if command in commands:
            commands[command]()
        elif command in commands_auth_code:
            validate_auth_code_flow_params(command)  # raises error in case one of the required params is missing
```